amato/umbrel-app.yml

```
@@ -0,0 +1,20 @@
+manifestVersion: 1
+id: amato
+category: productivity
+name: Amato Timer
+version: "0.0.1.1"
+tagline: Simple work timer.
+description: >-
+  A simple timer with default 25 minute increments to track your work.
+releaseNotes: >-
+  This is an initial alpha version
```

You can leave this empty as `releaseNotes` are shown for updates.

```suggestion
  releaseNotes: ""
```